spec/models/env_var_spec.rb

```diff
@@ -1,18 +1,15 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
+require './spec/support/main_helper'
+
```

Comment:
I thought this PR was about bridges? What are you doing in these specs? I know you are cleaning them up. In the future, keep your PR's focused. 

Make a pr about bridges => notice some cleaning up you can do, only clean up bridges, ignore the rest => merge bridges => create pr about cleaning up other code.